src/server/index.js

```diff
@@ -23,109 +24,134 @@ import './init';
 // Allow env config to run before importing holocron. (x2)
 import ssrServer from './ssrServer';
 import metricsServer from './metricsServer';
-import listen from './listen';
 import { addServer, shutdown } from './shutdown';
 import pollModuleMap from './utils/pollModuleMap';
 import loadModules from './utils/loadModules';
+import getHttpsConfig from './utils/getHttpsConfig';
 
-function ssrServerStart() {
+/*    🛠️ UTILITIES 🛠️    */
+
+const getRemotesFromDevEndpointsFile = () => {
+  const pathToDevEndpoints = path.join(process.cwd(), '.dev', 'endpoints', 'index.js');
+
+  return fs.existsSync(pathToDevEndpoints)
+    // eslint-disable-next-line global-require,import/no-dynamic-require
+    ? Object.values(require(pathToDevEndpoints)()).reduce(
+      (moduleRemotes, { destination, devProxyPath }) => ({
+        ...moduleRemotes,
+        [devProxyPath]: destination,
+      }),
+      {})
+    : {};
+};
+
+export const listen = async ({
```

Comment:
are you planning on moving back to `./listen`

Reply:
I am not since this is a fn only consumed in `index.js`